<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="8" column="1">
       <widget class="QSpinBox" name="sb_speed">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="11" column="0">
       <widget class="QPushButton" name="btn_stop_heat">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string> Отключить нагрев</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::DialogError"/>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_output">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="btn_start">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string> Start</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_sample">
        <property name="text">
         <string>Sample:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_operator">
        <property name="text">
         <string>Operator: </string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Скорость нагрева, мВ/с</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_calibration">
        <property name="text">
         <string>Calibration polynom</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QPushButton" name="btn_stop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string> Stop</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::MediaPlaybackStop"/>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_input">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Input:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="btn_output">
        <property name="text">
         <string>Output:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="plot_layout"/>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Измерение</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_save"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_new">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="action_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_cal">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
   </property>
   <property name="text">
    <string>Калибровка</string>
   </property>
  </action>
  <action name="action_close">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::WindowClose"/>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
